### ISSUE TITLE
[BUG] Multiblocks unforming when operation starts

### ISSUE THREAD

**Issue by QuantumAtomBreak** [zmaster587/AdvancedRocketry#2229]
[Please fill out the form below and delete the sections in square brackets after reading them]

## Version of Advanced Rocketry
>[2.0.0-16 and 2.0.0-10,11,12

>Why is this important?
>Knowing the version of Advanced Rocketry helps us narrow down an offending piece of code faster because we can easily determine what code is added and when.]

## Have you verified this is an issue in the latest unstable build
- [ ] Y
- [ ] N
- [x] N/A

>[Why is this important?
>If you're not testing with the latest build, you may be encountering a bug that's already been fixed.
>Reporting bugs that have already been fixed wastes everyone's time.  (please look in open issues too, somebody else could have already reported it!)]

## Version of LibVulpes
>[1.0.2-27

>Why is this important?
>Advanced rocketry depends on a library mod (LibVulpes), some errors can really be in that mod.  So knowing the verison helps us not only start looking for fixes there without you needing to anwer more questions, it also helps us know if you have a version mismatch.]

## Version of Minecraft
>[1.16.5

>Why is this important?
>Different versions of minecraft have different codebases for Advanced Rocketry.  Knowing the version of minecraft (and thus AR) helps us pinpoint issues quicker!]

## Does this occur without other mods installed
- [x] Y
>If Y, what is the MINIMUM set of mods required.
 Lib Vulpes, Advanced Rocketry and JEI (for testing convenience)
>[Why is this important?  
>Sometimes interactions between mods cause issues.  While it can be a problem with Advanced Rocketry, sometimes it's not.  When there are a lot of mods, it can take bugtesters HOURS to go through the list to determine the root cause.  When there are a lot of bug reports, that can add DAYS to the time it takes to fix a large set of issues.]

## Crash report or log or visualVM (if applicable)
http://pastebin.com is a good place to put them
crash reports that are put in the issue itself are
hard to read
>[insert here

> If it's an issue involving lag, please submit a visualVM sampler snapshot.  Mods like lag goggles often do not provide the kind of information needed to debug lag.
> For more information on how to create a snapshot, please see "How to Run CPU Sampling" [here](http://greyfocus.com/2016/05/visualvm-sampling/).  Then export the snapshot as an .nps.  You may have to zip it since github is weird, then please upload it here.

>Why is this important?
>Full crashlogs contain valuable information like where exactly in the code the error occurred, what other mods may be running and what version of forge you have.  This helps us replicate and diagnose the issue FAR faster.]

## Description of the problem
>[After assembling a multiblock (personally tested with rolling machine, lathe, precision assember, crystalliser and electric arc furnace) provide power and the resources for any recipe and enter the main GUI, click the I/O button to switch the machine on and it will immediately close the GUI and the formed mulktiblock structure will revert to the component blocks

>Why is this important?
>If we cannot reproduce the problem, we probably cannot fix it at all.]

**Comment by Misaka10034** [comment]
i got same problem,is there any solution?

**Comment by MightBeAHuman** [comment]
I've got this problem too, is it already solved in the newest version? Or are you guys still working on it?

**Comment by SPS360G** [comment]
im getting the problem too

**Comment by NathanDSimon** [comment]
Yep, same here. Annoyingly it also deletes your resources when this happens, I had to restore a backup because of this after losing my MekaSuit chestplate :( 
Fortunately the backup was recent enough that I didn't go back on too much progress, still frustrating though

**Comment by NathanDSimon** [comment]
Checking my logs, it appears something is wrong with the recipe manager; check the recipesmachine class...
```
[16:26:27] [Worker-Main-4/WARN]: Cannot add recipe!
[16:26:27] [Worker-Main-4/WARN]: null
	zmaster587.libVulpes.recipe.RecipesMachine.addRecipe(RecipesMachine.java:324)
	zmaster587.libVulpes.recipe.RecipesMachine.addRecipe(RecipesMachine.java:397)
	zmaster587.advancedRocketry.tile.multiblock.machine.TileChemicalReactor.registerRecipes(TileChemicalReactor.java:158)
	zmaster587.advancedRocketry.util.RecipeHandler.registerAllMachineRecipes(RecipeHandler.java:36)
	zmaster587.advancedRocketry.AdvancedRocketry.load(AdvancedRocketry.java:443)
	net.minecraftforge.eventbus.ASMEventHandler_1164_AdvancedRocketry_load_FMLCommonSetupEvent.invoke(.dynamic)
	net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:85)
	net.minecraftforge.eventbus.EventBus$$Lambda$2441/1622230027.invoke(Unknown Source)
	net.minecraftforge.eventbus.EventBus.post(EventBus.java:302)
	net.minecraftforge.eventbus.EventBus.post(EventBus.java:283)
	net.minecraftforge.fml.javafmlmod.FMLModContainer.acceptEvent(FMLModContainer.java:120)
	net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$4(ModContainer.java:121)
	net.minecraftforge.fml.ModContainer$$Lambda$3979/1651924497.run(Unknown Source)
	java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1618)
	java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1610)
	java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1689)
	java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
```
Unfortunately due to how you're handling the logging I can't actually tell which catch block you're falling to, so I can't tell what the exception actually is since the getLocalizedMessage() is returning null for your caught exception.
You may want to use e.toString() to manually get a string representation, or just hard-code it into the catch block.

**Comment by rapidfire9656** [comment]
Im having this same issue too.

**Comment by Pepelespooder** [comment]
Can confirm Same recipe Issues on my log Was wondering what was going on . Was gonna create an issue until i saw this

**Comment by psihopat93** [comment]
same problem! There is a solution?

**Comment by MightBeAHuman** [comment]
Is it already fixed? I really need a working version for 1.16...

**Comment by voidsong-dragonfly** [comment]
# It's almost like the mod is in _alpha_ or something, isn't it?
